ui/packages/app/src/pages/project/Project.js

```diff
@@ -74,7 +37,7 @@ const Project = () => {
   return (
     <EuiPageTemplate panelled={false} restrictWidth="90%">
       <EuiPageTemplate.Section>
-        {apps && !!currentProject ? (
+        {!!currentProject ? (
```

Comment:
Just curious, would anything happen if `apps` was empty? Or was that check redundant?

Reply:
It's simply redundant, it wasn't doing anything useful